CIFAR10/inference/Baidu_USA_GAIT_LEOPARD_team.txt

```diff
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": Baidu USA GAIT LEOPARD team: Baopu Li, Jiazhuo Wang, Zhiyu Cheng, Haofeng Kou, Yingze Bao, Xin Zhou
+	    "authorEmail": "baopuli@baidu.com", ìjiazhuowang@baidu.comî, ìzhiyucheng@baidu.comî, ìhaofengkou@baidu.comî, ìyingzebao@baidu.comî, ìzhouxin16@baidu.comî 
+	    "framework": "PyTorch v1.0.0.dev20181116",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-ResNet18",
```

Comment:
Please add the commit hash either to the `codeURL` or as a separate `commitHash` field